pkg/controller/storagecluster/reconcile.go

```diff
@@ -54,8 +54,8 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 
 	// ReconcileStrategyUnknown is the same as default
```

Comment:
Does this mean that ReconcileStrategyUnknown is now the same as ReconcileStrategyManage?